dynamics-nav/SETSELECTIONFILTER-Function.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "SETSELECTIONFILTER Function"
+description: SETSELECTIONFILTER Function Notes the records that the user has selected on the page, marks those records in the table specified, and sets the filter to marked only.
```

Comment:
Too long per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.